apps/channels/tests/model_manager_test.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from django.test import TestCase
+
+from apps.channels.models import ChannelPlatform, ExperimentChannel
+from apps.experiments.models import ConsentForm, Experiment, Prompt
+from apps.service_providers.models import LlmProvider
+from apps.teams.models import Team
+from apps.users.models import CustomUser
+
+
+class TestExperimentChannelObjectManager(TestCase):
```

Comment:
no need to change this now since you've already written it but I'd encourage you to look into `pytest`. This is a good article to start with: https://djangostars.com/blog/django-pytest-testing/